src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,28 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
```

Comment:
we never get health as an argument, it'd be better to move this inside the constructor

Reply:
It was ok too 🙂